test/order_test.rb

```diff
@@ -60,102 +60,149 @@
       end
     end
   end
-
+  
   describe "#total" do
     it "Returns the total from the collection of products" do
       products = { "banana" => 1.99, "cracker" => 3.00 }
       order = Order.new(1337, products, customer)
-
+      
       expected_total = 5.36
-
+      
       expect(order.total).must_equal expected_total
     end
-
+    
     it "Returns a total of zero if there are no products" do
       order = Order.new(1337, {}, customer)
-
+      
       expect(order.total).must_equal 0
     end
   end
-
+  
   describe "#add_product" do
     it "Increases the number of products" do
       products = { "banana" => 1.99, "cracker" => 3.00 }
       before_count = products.count
       order = Order.new(1337, products, customer)
-
+      
       order.add_product("salad", 4.25)
       expected_count = before_count + 1
       expect(order.products.count).must_equal expected_count
     end
-
+    
     it "Is added to the collection of products" do
       products = { "banana" => 1.99, "cracker" => 3.00 }
       order = Order.new(1337, products, customer)
-
+      
       order.add_product("sandwich", 4.25)
       expect(order.products.include?("sandwich")).must_equal true
     end
-
+    
     it "Raises an ArgumentError if the product is already present" do
       products = { "banana" => 1.99, "cracker" => 3.00 }
-
+      
       order = Order.new(1337, products, customer)
       before_total = order.total
-
+      
       expect {
         order.add_product("banana", 4.25)
       }.must_raise ArgumentError
-
+      
       # The list of products should not have been modified
       expect(order.total).must_equal before_total
     end
   end
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
-    end
-
-    it "Returns accurate information about the first order" do
```

Comment:
Your indentation is kind of weird here and is why Github thought you deleted this test.  (It looks like you moved some tests outside of `describe`s.)

Some sort of formatter for VS Code could help you with noticing changes like this.